tests/unit/data/sparkle/sparkle1_html_parser_result.json

```diff
@@ -0,0 +1,44 @@
+[
+    {
+        "account": "Not Available",
+        "circuits": [
+            {
+                "circuit_id": "CIR0000000000001",
+                "impact": "OUTAGE"
+            }
+        ],
+        "end": 1628593200,
+        "maintenance_id": "1111111",
+        "start": 1628564400,
+        "status": "COMPLETED",
+        "summary": "A scheduled work will be carried out to perform hardware upgrade in order in order to ensure the continued integrity of the network."
+    },
+    {
+        "account": "Not Available",
+        "circuits": [
+            {
+                "circuit_id": "CIR0000000000001",
+                "impact": "OUTAGE"
+            }
+        ],
+        "end": 1628679600,
+        "maintenance_id": "22222",
+        "start": 1628650800,
+        "status": "COMPLETED",
+        "summary": "A scheduled work will be carried out to perform hardware upgrade in order in order to ensure the continued integrity of the network."
+    },
+    {
+        "account": "Not Available",
+        "circuits": [
+            {
+                "circuit_id": "CIR0000000000001",
+                "impact": "OUTAGE"
+            }
+        ],
+        "end": 1628766000,
+        "maintenance_id": "33333",
+        "start": 1628737200,
+        "status": "COMPLETED",
+        "summary": "A scheduled work will be carried out to perform hardware upgrade in order in order to ensure the continued integrity of the network."
+    }
```

Comment:
Just to confirm I understand correctly  - for Sparkle, we are potentially getting multiple distinct maintenance_ids for the same circuit? Not multiple circuits for a single maintenance_id?

Reply:
For the same circuit. The problem with this is that we only have one example to go off. There could be other variations of this.